asserts/ifacedecls.go

```diff
@@ -434,10 +441,10 @@ func compilePlugRule(interfaceName string, rule interface{}) (*PlugRule, error)
 		return nil, err
 	}
 	if rMap == nil {
-		if shortcut {
-			rMap = defaultAllowRule
-		} else {
-			rMap = denyRule
+		if shortcut == "true" {
+			rMap = defaultOutcome
+		} else { // "false"
```

Comment:
I had the impression that with shortcut bool, it wasn't clear whether it was carrying the shortcut value or telling there was shortcut